src/Blueprint/Blueprint/Cmdlets/GetAzureRMBlueprintAssignment.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.Blueprint.Common;
+using ParameterSetNames = Microsoft.Azure.Commands.Blueprint.Common.BlueprintConstants.ParameterSetNames;
+using ParameterHelpMessages = Microsoft.Azure.Commands.Blueprint.Common.BlueprintConstants.ParameterHelpMessages;
+
+namespace Microsoft.Azure.Commands.Blueprint.Cmdlets
+{
+    [Cmdlet(VerbsCommon.Get, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "BlueprintAssignment")]
+    public class GetAzureRmBlueprintAssignment : BlueprintCmdletBase
+    {
+        #region Parameters
+
+        [Parameter(ParameterSetName = ParameterSetNames.BlueprintAssignmentByName, Position = 0, Mandatory = false, ValueFromPipelineByPropertyName = true, HelpMessage = ParameterHelpMessages.SubscriptionId)]
+        [Parameter(ParameterSetName = ParameterSetNames.BlueprintAssignmentsBySubscription, Position = 0, Mandatory = false, ValueFromPipelineByPropertyName = true, HelpMessage = ParameterHelpMessages.SubscriptionId)]
+        [ValidateNotNullOrEmpty]
+        public string SubscriptionId { get; set; }
+
+        [Parameter(ParameterSetName = ParameterSetNames.BlueprintAssignmentByName, Position = 1, Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = ParameterHelpMessages.BlueprintAssignmentName)]
+        [ValidateNotNullOrEmpty]
+        public string Name { get; set; }
+        #endregion Parameters
+
+        #region Cmdlet Overrides
+        public override void ExecuteCmdlet()
+        {
+            var subscription = SubscriptionId ?? DefaultContext.Subscription.Id;
+
+            try
+            {
+                switch (ParameterSetName) {
+                    case ParameterSetNames.BlueprintAssignmentsBySubscription:
+                        foreach (var assignment in BlueprintClient.ListBlueprintAssignments(string.Format(BlueprintConstants.SubscriptionScope, subscription)))
```

Comment:
>string.Format(BlueprintConstants.SubscriptionScope, subscription) [](start = 92, length = 65)

This sort of thing repeats a lot, so it probably should be a utility function somewhere.